libs/langchain/langchain/agents/agent_toolkits/openapi/base.py

```diff
@@ -30,7 +30,20 @@ def create_openapi_agent(
     agent_executor_kwargs: Optional[Dict[str, Any]] = None,
     **kwargs: Any,
 ) -> AgentExecutor:
-    """Construct an OpenAPI agent from an LLM and tools."""
+    """Construct an OpenAPI agent from an LLM and tools.
+
+    *Security Note*: When creating an OpenAPI agent, check the permissions
+        and capabilities of the underlying toolkit.
+
+        For example, if the default implementation of OpenAPIToolkit
+        uses the RequestsToolkit which contains tools to make arbitrary
+        network requests against any URL (e.g., GET, POST, PATCH, PUT, DELETE),
```

Comment:
This seems like a sentence fragment, possibly some text is missing? Or did you mean to end it with a period?